lib/rubocop/cop/lint/boolean_symbol.rb

```diff
@@ -27,11 +27,27 @@ class BooleanSymbol < Cop
 
         def_node_matcher :boolean_symbol?, '(sym {:true :false})'
 
+        def_node_matcher :boolean_hash?, '(pair (sym {:true :false}) _)'
+
         def on_sym(node)
           return unless boolean_symbol?(node)
 
           add_offense(node, message: format(MSG, boolean: node.value))
         end
+
+        def autocorrect(node)
+          lambda do |corrector|
+            replacement = node.source.delete(':')
```

Comment:
I think it will be clearer if this lvar naming is the following name.

```suggestion
            boolean_literal = node.source.delete(':')
```